Main.java

```diff
@@ -1,8 +1,8 @@
-package com.company;
-
 public class Main {
 
     public static void main(String[] args) {
+        String MyName = "DAngelo";
+        System.out.println (MyName);
 
```

Comment:
Good job.  Normal naming convention for variable is to start the first word with lowercase and the second word with an uppercase letter. Normally you wouldn't have a space between println and (), but java ignores whitespace, so it is valid.  Those are minor things though good job overall.